examples/causal_inference/interventional_distribution.ipynb

```diff
@@ -0,0 +1,1561 @@
+{
```

Comment:
<p>Maybe "The joint distribution could be described as: " -&gt; "The joint distribution can be factorized (or expressed) as: "</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/552/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Thanks for keeping me not lazy. I've added more detail about factorisation and truncated factorisation</p><p> </p>

Reply:
<p>Yeah, i like the emphasis on factorized or decomposed given the uses of the joint distributions we're going to see below. </p>